### DISCUSSION TITLE
Is anyone interested in a privacy-focused Google tag manager alternative?

### DISCUSSION THREAD

**Discussion by LinusGeffarth** [plausible/analytics#1651] [Anything Else]
<sub>[with permission from plausible, I'm reaching out to y'all because this a privacy-focused community]</sub>

## TL;DR:
Is anyone here interested in a simple, pretty, privacy-focused alternative to Google tag manager?

And if yes, would you be willing to pay for example 2€/site/month?
Or would you prefer flat pricing, such as 10€/month for any number of sites?
More? Less?

Thank you for your thoughts 🙏
Linus

---

Why? 🤔

Yesterday, I wanted to add a script tag to my web-app. My repository is huge and I work with other devs; we coordinate releases on a monthly basis.
Luckily, we have Google tag manager (GTM) installed so I can install the script immediately and right from my iPad.

As I fired up GTM, I found it yet again frustratingly complicated with way too many features I've never used, even though I've been a user for years(!).
After I managed to submit the new tag, I realized: wait a minute, I'm from the EU, I think I'm not even allowed to use GTM, afaik. What now? Right:

Build my own; inspired by plausible's story.
But before I get all hyped about yet another side-project, I wanted to see whether there's even a need for such a project.

So, is anyone here interested in a simple, pretty, privacy-focused alternative to Google tag manager?

And if yes, would you be willing to pay for example 2€/site/month?
Or would you prefer flat pricing, such as 10€/month for any number of sites?
More? Less?

In any case, I'd greatly appreciate your thoughts!
Cheers to my fellow makers, Linus 🙋‍♂️

**Comment by Johannes5** [comment]
yeah, I'm looking for a GTM alternative

**Reply by Johannes5** [reply]
I'd pay 2€ per domain per month

the main purpose would be to turn tags on/off based on events.

**Reply by LinusGeffarth** [reply]
Thanks!

What kind of events do you need?

**Reply by Johannes5** [reply]
when the user disagrees to cookie tracking

**Reply by Johannes5** [reply]
I told myself not to use any google products in my stack, but since I currently lack an alternative: is GTM actually compromising the data of my visitors? If so how exactly?
And I've never heard of GTM being verboten in the EU.

**Comment by masteradhoc** [comment]
Yeah definately!
